Dockerfile-multistage

```diff
@@ -1,68 +1,52 @@
 ARG 	CHEMISTRY_PACKAGE=bbchem
-FROM maven:3-openjdk-8 as builder
+FROM maven:3-openjdk-11 as builder
 
 FROM 	builder as dependencies
 ARG     CHEMISTRY_PACKAGE
 ENV     CHEMISTRY_PACKAGE=${CHEMISTRY_PACKAGE}
 
 FROM 	dependencies as jchem
-COPY 	lib/jchem-16.4.25.0.jar /lib/jchem-16.4.25.0.jar
-RUN   mvn install:install-file -Dfile=/lib/jchem-16.4.25.0.jar -DartifactId=jchem -DgroupId=com.chemaxon -Dversion=16.4.25.0 -Dpackaging=jar -DgeneratePom=true -DcreateChecksum=true
+ADD settings.xml /root/.m2/settings.xml
 
 FROM 	dependencies as bbchem
 
 FROM 	dependencies as indigo
 
 FROM 	${CHEMISTRY_PACKAGE} as compile
-
-COPY 	pom.xml /src/pom.xml
+ADD 	pom.xml /src/pom.xml
 WORKDIR /src
-RUN 	--mount=type=cache,target=/root/.m2 mvn dependency:resolve -P ${CHEMISTRY_PACKAGE}
+RUN 	mvn dependency:resolve -P ${CHEMISTRY_PACKAGE}
+# Caching doesn't seem to work with the ChemAxon maven repo
+# RUN 	--mount=type=cache,target=/root/.m2 mvn dependency:resolve -P ${CHEMISTRY_PACKAGE}
```

Comment:
This file generally worked before and definitely sped up the use case of speeding up indigo and bbchem chemistry packages before.  We need to maintain that speed.  Can we ever these cache changes? 

There is a separate file that needs to get updated called Dockerfile-jchem.  I think you should update that file with the equivalent changes made for jchem in this file, and then just use the Dockerfile-jchem to build the docker image.